Tests/MongoSwiftSyncTests/SpecTestRunner/FailPoint.swift

```diff
@@ -0,0 +1,127 @@
+import MongoSwiftSync
```

Comment:
While I was moving test files over, it seemed like this was worthy of its own file.